README.md

```diff
@@ -121,7 +126,8 @@ a(); // Object -- so far so good.
 a().getA(); // "a"
 ```
 
-Yes. Got it. In both of these instances, we actually created a brand new object, and then immediately threw it away, because we didn't assign it to anything. Don't worry about that.
+Yes. Got it. In both of these instances, we actually created a brand new object, and then immediately threw it away, 
+because we didn't assign it to anything. Don't worry about that.
```

Comment:
This is Markdown. The idea behind it is to be readable both as raw text and as a rendered page. Read first sentence here: http://daringfireball.net/projects/markdown/

A long single line treated the same as the multiple lines - as a single paragraph `<p></p>`. Unless there are two line breaks `\n\n`, which are treated as the paragraph end and a new paragraph start `</p><p>`. Read first sentences here: http://daringfireball.net/projects/markdown/syntax#p

Having a long 200 characters line is really hard to read raw (i.e. in an IDE). And most text editors do not have word wrapping enabled by default.